### ISSUE TITLE
I Came across These Errors, however It Appears that There Are No porting.info.gz Files Available

### ISSUE THREAD

**Issue by P0W3** [msys2/MINGW-packages#18557]
I came across these errors.

![image](https://github.com/msys2/MSYS2-packages/assets/18279115/25747f06-2942-4312-8ced-9dba162e1ba2)

```
error: failed to commit transaction (conflicting files)
/clang64/share/info/porting.info.gz exists in both 'mingw-w64-clang-x86_64-arm-none-eabi-newlib' and 'mingw-w64-clang-x86_64-riscv64-unknown-elf-newlib'
/ucrt64/share/info/porting.info.gz exists in both 'mingw-w64-ucrt-x86_64-arm-none-eabi-newlib' and 'mingw-w64-ucrt-x86_64-riscv64-unknown-elf-newlib'
/mingw64/share/info/porting.info.gz exists in both 'mingw-w64-x86_64-arm-none-eabi-newlib' and 'mingw-w64-x86_64-riscv64-unknown-elf-newlib'
Errors occurred, no packages were upgraded.
```

However, it appears that there are no porting.info.gz files available.

![image](https://github.com/msys2/MSYS2-packages/assets/18279115/60dfaf26-43eb-4372-be1f-8c463ec8d694)
![image](https://github.com/msys2/MSYS2-packages/assets/18279115/427af06a-a782-4aa9-b9fe-cf68ac33b32c)
![image](https://github.com/msys2/MSYS2-packages/assets/18279115/a455ed15-4795-4ed2-aac6-0d76ffa19d09)

**Comment by lazka** [comment]
@P0W3 can you try again now? there have been some related fixes for those packages

**Comment by Biswa96** [comment]
This issue was fixed in 2d3b20558e637b85601f6363d5063d81ad36fc69 and 5124b56546459a77c623dc09e1cd996ead723972 commits.
